src/util.py

```diff
@@ -4,8 +4,21 @@
 import os
 import pickle
 
+# Standard library imports
+import os
+from functools import reduce
+os.environ['OUTDATED_IGNORE'] = '1'
```

Comment:
Just curious, what is this for?

Reply:
it will ignore the outdated package reminder. E.g., xx latest version is 2.0.0 but your current is 1.5.3

Reply:
os.environ['OUTDATED_IGNORE'] = '1' would ignore message indicate which packages is outdated. E.g., xxx latest version is 2.0.0, your current version is 1.5.2